src/views/SystemProfile/components/SubSystems/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,240 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`System Sub-systems subpage matches snapshot 1`] = `
+<DocumentFragment>
+  <div
+    id="system-sub-systems"
+  >
+    <div
+      class="grid-container padding-left-0 padding-right-0"
+      data-testid="gridContainer"
+    >
+      <div
+        class="grid-row grid-gap"
+        data-testid="grid"
+      >
+        <div
+          class="desktop:grid-col-8"
+          data-testid="grid"
+        >
+          <h2
+            class="margin-top-0 margin-bottom-4"
+          >
+            Sub-systems
+          </h2>
+          <ul
+            class="usa-card-group margin-0"
+            data-testid="CardGroup"
+          >
+            <li
+              class="usa-card grid-col-12 margin-bottom-2"
+              data-testid="Card"
+            >
+              <div
+                class="usa-card__container"
+              >
+                <header
+                  class="usa-card__header padding-2 padding-bottom-0"
+                  data-testid="CardHeader"
+                >
+                  <div
+                    class="display-flex margin-bottom-105"
+                  >
+                    <div
+                      class="flex-fill"
+                    >
+                      <h3
+                        class="margin-y-0 line-height-body-2"
+                      >
+                        <a
+                          class="usa-link"
+                          href="/system-profile/326-9-0/todo"
```

Comment:
Because this 'todo' component doesn't exist, it will crash the app trying to render.  We should have a check on url parameters before attempting to render.

In SystemProfile.tsx we can add this check on line 149 to ensure component exists.

`  // TODO: Handle errors and loading
  if (error || !systemInfo || (subinfo && !sideNavItems(systemInfo)[subinfo])) {
    return <NotFound />;
  }`